src/model_diagnostics/__about__.py

```diff
@@ -14,4 +14,4 @@
 # Dev branch marker is: 'X.Y.dev' or 'X.Y.devN' where N is an integer.
 # 'X.Y.dev0' is the canonical version of 'X.Y.dev'
 #
-__version__ = "1.0.1"
+__version__ = "1.0.2"
```

Comment:
Please revert.

Reply:
Done in 2302828ad62e11177f00f762483d5f4b9a627cf3
I close this